### PR TITLE
EF Core series ownership and rp-over-mvc include file

### PR DESCRIPTION
**Pull request by tdykstra**

### CODE REVIEW

File: aspnetcore/includes/RP-EF/rp-over-mvc.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-* Is more current with the latest API.
-* Covers more features.
+Some things this MVC tutorial has that the Razor Pages tutorial doesn't:
+
+* Implement inheritance in the data model
+* Perform raw SQL queries
+* Use dynamic LINQ to simplify code
+ 
+Some things the Razor Pages tutorial has that this one doesn't:
+
```
Comment (Rick-Anderson) [inline aspnetcore/includes/RP-EF/rp-over-mvc.md]: * Provides more EF Core best practices.	
 * Uses more efficient queries.	
* Implement inheritance in the data model

Comment (Rick-Anderson) [reply aspnetcore/includes/RP-EF/rp-over-mvc.md]: The EF folks reviewed the RP version (which was a straight port of the MVC version) and made a bunch of changes that never made it back to the MVC version.

Comment (tdykstra) [reply aspnetcore/includes/RP-EF/rp-over-mvc.md]: On best practices and efficient queries, I retrofitted EF MVC with the ones I found while I was updating to 2.2 (e.g., FirstOrDefault).  The only one I'm aware of that I didn't retrofit was Select for related data.  What specifically did I miss?

Comment (Rick-Anderson) [reply aspnetcore/includes/RP-EF/rp-over-mvc.md]: I don't remember, but I thought there were a few more. Go ahead.

**Pull request merged**